UPDATING.md

```diff
@@ -41,6 +41,10 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+### Fernet is enabled by default
```

Comment:
Can we make it more descriptive, please?

- Previous Behaviour
- Changed Behavior
- Reason of the change